### ISSUE TITLE
Release a hotfix version to make it compilable with Xcode 13

### ISSUE THREAD

**Issue by AF-cgi** [SwiftGen/SwiftGen#878]
Please merge the open PRs to release a hotfix version. Currently it's not possible to compile SwiftGen with Xcode 13. From that reason It's not possible to install it via brew or Mint, if you have only Xcode 13 installed.

**Comment by djbe** [comment]
Merged a fix into `develop`, for now you can point both homebrew & mint to that branch and it should work 🤞 Should help until we create an actual release.

**Comment by ChristopherNorthcube** [comment]
When running `develop` I get this issue:

SwiftGenCLI/resource_bundle_accessor.swift:11: Fatal error: could not load resource bundle: from /var/root/.mint/packages/github.com_SwiftGen_SwiftGen/build/develop/SwiftGen_SwiftGenCLI.bundle or /private/var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/mint/github.com_SwiftGen_SwiftGen/.build/x86_64-apple-macosx/release/SwiftGen_SwiftGenCLI.bundle

**Comment by 4brunu** [comment]
I have the same issue

**Comment by lfcj** [comment]
I am seeing:
```
'Kanna' /private/var/folders/zz/j434sz1d6tnd5jwy50dnzvhc0000gp/T/mint/github.com_SwiftGen_SwiftGen/.build/checkouts/Kanna: warning: Invalid Exclude '/private/var/folders/zz/j434sz1d6tnd5jwy50dnzvhc0000gp/T/mint/github.com_SwiftGen_SwiftGen/.build/checkouts/Kanna/Sources/Tests/KannaTests/Data': File not found.
error	28-Sep-2021 07:39:19	'Kanna' /private/var/folders/zz/j434sz1d6tnd5jwy50dnzvhc0000gp/T/mint/github.com_SwiftGen_SwiftGen/.build/checkouts/Kanna: warning: Invalid Exclude '/private/var/folders/zz/j434sz1d6tnd5jwy50dnzvhc0000gp/T/mint/github.com_SwiftGen_SwiftGen/.build/checkouts/Kanna/Sources/Sources/Info.plist': File not found.
error	28-Sep-2021 07:39:19	'Kanna' /private/var/folders/zz/j434sz1d6tnd5jwy50dnzvhc0000gp/T/mint/github.com_SwiftGen_SwiftGen/.build/checkouts/Kanna: warning: Invalid Exclude '/private/var/folders/zz/j434sz1d6tnd5jwy50dnzvhc0000gp/T/mint/github.com_SwiftGen_SwiftGen/.build/checkouts/Kanna/Sources/Sources/Kanna.h': File not found.
error	28-Sep-2021 07:39:19	🌱 Encountered error during "swift build -c release -Xswiftc -target -Xswiftc x86_64-apple-macosx11.5". Use --verbose to see full output
error	28-Sep-2021 07:39:19	🌱  Failed to build SwiftGen 6.4.0 with SPM
```

with `SwiftGen/SwiftGen@6.4.0`. Using `SwiftGen/SwiftGen@develop` worked, though.

**Comment by 4brunu** [comment]
I was also able to install `SwiftGen/SwiftGen@develop` with success, but when I run it I got the following error: 
```
mint run swiftgen config run --config .swiftgen.yml
SwiftGenCLI/resource_bundle_accessor.swift:11: Fatal error: could not load resource bundle: from /Users/user/.mint/packages/github.com_SwiftGen_SwiftGen/build/develop/SwiftGen_SwiftGenCLI.bundle or /private/var/folders/qw/ymw6c9d14kv4k4dr2yt_q2xh0000gn/T/mint/github.com_SwiftGen_SwiftGen/.build/x86_64-apple-macosx/release/SwiftGen_SwiftGenCLI.bundle
zsh: illegal hardware instruction  mint run swiftgen config run --config .swiftgen.yml
```

**Comment by lfcj** [comment]
@4brunu, what solved that for me was cleaning the Mint cache and installing again:
```
rm -rf ~/.mint/metadata.json
rm -rf ~/.mint/packages
```
(I did not do `rm -rf ~/.mint` out of lazyness in case Mint would not create the folder again)
Hope it helps.

**Comment by 4brunu** [comment]
That didn't work for me, I still have the same issue.

**Comment by mthole** [comment]
Quick investigation of this:

The `develop` branch's Package.swift is here: https://github.com/SwiftGen/SwiftGen/blob/develop/Package.swift

Notice that the `swiftgen` executable depends on `SwiftGenCLI`, which (new in `develop`) includes a `resources` phase for copying templates. I suspect that this resource bundle lookup is being triggered and failing when the executable is run, leading to the problem we're seeing here.

**Comment by mthole** [comment]
A quick-and-dirty fix for anyone else running into this via a `Mintfile` installation, you can point to my fork:
`mthole/SwiftGen@xc13-fix`

See https://github.com/SwiftGen/SwiftGen/compare/stable...mthole:xc13-fix?expand=1 for the diff

**Comment by 4brunu** [comment]
Could you please open a PR?

**Comment by djbe** [comment]
Implemented in SwiftGen 6.5.0, which has been released 🎉
